sno/cli.py

```diff
@@ -94,8 +111,15 @@ def print_version(ctx):
     "--version", callback=print_version, help="Show version information and exit.",
 )
 @click.option("-v", "--verbose", count=True, help="Repeat for more verbosity")
+# NOTE: this option isn't used in `cli`, but it is used in `PdbGroup` above.
+@click.option(
+    "--post-mortem",
```

Comment:
wonder if we should have a consistent prefix for experimental/internal options? eg: `--x-post-mortem`

Reply:
IMO, not required. It's not like we're going to have a lot of them, and `hidden=True` already implicitly means we can break them when we want to.